README.md

```diff
@@ -31,6 +47,17 @@ C++ Idiomatic Clients for [Google Cloud Platform][cloud-platform] services.
 [kokoro-noex-link]: https://storage.googleapis.com/cloud-cpp-kokoro-status/kokoro-noex-link.html
 [kokoro-centos-7-shield]: https://storage.googleapis.com/cloud-cpp-kokoro-status/kokoro-centos-7.svg
 [kokoro-centos-7-link]: https://storage.googleapis.com/cloud-cpp-kokoro-status/kokoro-centos-7-link.html
+[kokoro-check-abi-shield]: https://storage.googleapis.com/cloud-cpp-kokoro-status/kokoro-check-abi.svg
```

Comment:
I don't see these producing any output when I click "View file".  Am I missing something?

Reply:
This is what I see:

![image](https://user-images.githubusercontent.com/6241635/55026360-33f98980-4fd9-11e9-9956-5a21a027bd77.png)

Do you see something different?

Reply:
I see that, but I don't see anything between the "Client Library Documentation" link and the "This library supports the following ..." text, where these sit.  Are they supposed to render something?

Reply:
Ah, those are link definitions (or reference-style links):

https://www.markdownguide.org/basic-syntax/#reference-style-links

basically something like `[foo]: https://example.com` defines `[foo]` as an alias for `https://example.com`. It leaves the inline links a little more readable.

Reply:
Gotcha.  Thanks.